content/applications/marketing/sms_marketing/essentials/mailing_lists_blacklists.rst

```diff
@@ -0,0 +1,146 @@
+   :align: center
+   :alt: View of the mailing lists contact page in the Odoo SMS Marketing application.
+
+.. note::
+   By default, Odoo reveals the :guilabel:`Mailing List Contacts` page with the :guilabel:`Exclude
+   Blacklisted Phone` filter in the search bar. Therefore only showing contact information for
+   recipients who still want to receive communications and mailings.
+
+Communication history in the Chatter
+------------------------------------
+
+An accessible record of the sent mailings is kept on each recipient's :guilabel:`Chatter`, located
+beneath a recipient's contact form (in the :guilabel:`Contacts` application).
+
+Database users can reference the :guilabel:`Chatter` to easily keep track of communications, and see
+a history of interactions with contacts and prospects.
+
+For example, sales representatives can use the :guilabel:`Chatter` to quickly find out which SMS
+mailing promotions a certain customer has received (or not received).
+
+.. image:: mailing_lists_blacklists/sms-marketing-chatter.png
+   :align: center
+   :alt: View of what the chatter looks like in the Odoo Contacts app.
+
+Blacklist
+=========
+
+:guilabel:`Odoo SMS Marketing` has a :guilabel:`Blacklist` feature that provides recipients with the
+power to add their phone number to a list of people who no longer want to receieve communications or
+mailings.
+
+This is also known as the "Unsubscribe Process." Customers will automatically be added onto the
+:guilabel:`Blacklist`, if they click :guilabel:`Unsubscribe`, via their :guilabel:`Subscription
+Management Page`. Customers can also be added manually to the blacklist, if necessary.
+
+To see a complete collection of blacklisted numbers, navigate to the :guilabel:`SMS Marketing` app,
+and go to :menuselection:`Configuration --> Blacklisted Phone Numbers`. Doing so, reveals a separate
+page containing every blacklisted phone number in the database.
+
+.. image:: mailing_lists_blacklists/sms-blacklist-menu.png
+   :align: center
+   :alt: SMS Blacklist menu in the application.
+
+To manually add a number, click the :guilabel:`Create` button in the upper-left. When clicked, Odoo
+reveals a separate page, in which the phone number to be blacklisted is entered. There's also a
+checkbox to indicate whether that particular phone numnber is :guilabel:`Active` (or not).
+
+.. image:: mailing_lists_blacklists/sms-blacklist-create.png
+   :align: center
+   :alt: SMS Blacklist menu in the application.
+
+Once the form is completed, click :guilabel:`Save` to add it to the :guilabel:`Blacklist`. To
+remove any number from the :guilabel:`Blacklist`, select the desired number, and click
+:guilabel:`Unblacklist`.
+
+Importing blacklists
+--------------------
+
+During a software/platform migration, it is possible to import an already existing blacklist of
+contacts. This would include customers, who have already asked to be :guilabel:`Blacklisted` on SMS
```

Comment:
```suggestion
contacts. This would include customers, who have already asked to be blacklisted` on :abbr:`SMS
(Short Message Service)` mailings.
```